packages/plugin-audio-button-response/src/index.ts

```diff
@@ -177,6 +183,7 @@ class AudioButtonResponsePlugin implements JsPsychPlugin<Info> {
       display_element.innerHTML = html;
 
       if (trial.response_allowed_while_playing) {
+        disable_buttons();
```

Comment:
Hi @thtTNT, I'm currently merging this into the refactored plugins on the `v8` branch and I'm wondering what the disabling and immediate re-enabling is for here?

Reply:
Got it, sorry for the confusion. I'll need to refactor this anyway to merge it into the new version, so I hope you don't mind if I change it a bit. Cheers!